src/tests/test_regionIDs.R

```diff
@@ -2,75 +2,106 @@
 #Load package testthat, and install it when this has not been done before
 #-------------------------------------------------------------------------
 
-packages <- c("testthat","sf","utils","readr")
+packages <- c("testthat","sf","utils","readr","here")
 
-for(i in packages) {
-  if( ! i %in% rownames(installed.packages()) ) { install.packages( i ) }
-  library(i, character.only = TRUE)
+for(package in packages) {
+  if( ! package %in% rownames(installed.packages()) ) { install.packages( package ) }
+  library(package, character.only = TRUE)
 }
 
 
 #-------------------------------------------------------------------------
-#Load the data of the datasets that you want to test
+#List the datasets to test
 #-------------------------------------------------------------------------
-points <- st_read("./data/output/UAT_processing/Vespa_velutina_shape/points.geojson")
-nesten<- st_read("./data/output/UAT_processing/Vespa_velutina_shape/nesten.geojson")
-aantal_gemelde_nesten<- st_read("./data/output/UAT_processing/Vespa_velutina_shape/aantal_gemelde_nesten.geojson")
-aantal_lente_nesten<- read.csv("./data/output/UAT_processing/Vespa_velutina_shape/aantal_lente_nesten.csv")
-actieve_haarden<- st_read("./data/output/UAT_processing/Vespa_velutina_shape/actieve_haarden.geojson")
-beheerde_nesten<- st_read("./data/output/UAT_processing/Vespa_velutina_shape/beheerde_nesten.geojson")
-onbehandelde_nesten<- st_read("./data/output/UAT_processing/Vespa_velutina_shape/onbehandelde_nesten.geojson")
+#Specify folders holding the datasets to test
+folders<-c("./data/output/UAT_processing/Vespa_velutina_shape",
+           "./data/output/UAT_direct")
 
+#List files in those folders
+datasets <- unlist(lapply(folders, function(folder) {
+  list.files(folder, pattern = "\\.(geojson|csv)$", full.names = TRUE)
+}))
 
-#-------------------------------------------------------------------------
-#Load the translations file
-#-------------------------------------------------------------------------
-translations_regions<-read.csv2("./data/output/UAT_direct/translations_regions.csv")
+#Remove certain files that don't need to be checked
+datasets<-datasets[!datasets %in% c("./data/output/UAT_direct/translations_simple.csv",
```

Comment:
add gemeentes.geojson en provincies.geojson

Reply:
I think it may be a good idea to have these columns in those files, as they are often used to append information to datasets

Reply:
hmm I follow you but only partly. 
Some arguments against adding all columns:

1. Adding gemeente to provincies.geojson makes little sense. 
2. Also currently gemeentes are limited to the provinces but in the future this will possibly no longer be the case. This will make this a complicated file to manage. 

Some arguments pro:
1. we would only need 1 spatial join instead of 2 (or more)

Reply:
Okay, for now I will exclude the provinces and communes from the test and then we can see in the future if there would be need to change that.